### PR TITLE
update login and register page

### LINKED ISSUE #1: Add login providers

### LINKED ISSUE #12: Update login and signup page

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
Fix #12 
Fix #1

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": false,
```
Comment (HarryET) [inline .prettierrc.json:4]: Semicolons should be true

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    "singleQuote": true,
```
Comment (HarryET) [inline .prettierrc.json:5]: Are we using single or double?

File: .prettierrc.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+    "trailingComma": "es5",
```
Comment (HarryET) [inline .prettierrc.json]: Were using es6 so this needs to be updated

File: tailwind.config.js [hunk 4]
```diff
@@ -4,8 +4,14 @@ module.exports = {
   theme: {
     extend: {
       colors: {
-        "supabase": "#24B47E"
```
Comment (HarryET) [inline tailwind.config.js:7]: Keep supabase as it could be used later

**Review by HarryET — CHANGES_REQUESTED** [review]
Just a few changes, there is also the mobile changes mentioned in discord then it should be good to merge

**Comment by VictorPeralta** [comment]
Great job!

Some things I changed, just to keep you in the loop:
- `emailErrorMessageForType` and `passwordErrorMessageForType` were being defined in `register.tsx` and `login.tsx`. I've deleted the register copies and exported the `login.tsx` versions. We could move these to a utils file later if we want to.
- Removed unused `watch` variable (also added linting rule to help us catch these)
- Added LoginIcons file, where social login SVGs are placed.

File: .prettierrc.json (hunk 1 above)
Comment (Isaiah-Hamilton) [reply .prettierrc.json:4]: as on discord I think semicolons should be false

File: .prettierrc.json (hunk 2 above)
Comment (Isaiah-Hamilton) [reply .prettierrc.json:5]: I like to use single but I'm with using ether one

File: .prettierrc.json (hunk 1 above)
Comment (HarryET) [reply .prettierrc.json:4]: 👌🏼

**Review by HarryET — DISMISSED** [review]

**Pull request merged**